meta-oe/recipes-extended/polkit/polkit-group-rule.inc

```diff
@@ -5,9 +5,3 @@ inherit features_check
 REQUIRED_DISTRO_FEATURES = "polkit"
 
 inherit useradd
```

Comment:
Below you remove the only useradd parameters, so there is no user created/modified anymore.

Therefore this inherit should be removed too, together with
recipes-extended/polkit/polkit-group-rule-datetime.bb:USERADD_PACKAGES = "${PN}"
recipes-extended/polkit/polkit-group-rule-network.bb:USERADD_PACKAGES = "${PN}"
recipes-extended/polkit/polkit-group-rule-udisks2.bb:USERADD_PACKAGES = "${PN}"

Reply:
Those recipes are adding new groups, so I think it's still needed? eg: `GROUPADD_PARAM:${PN} = "--system network"`

Reply:
OK, sorry for the noise.